src/App.tsx

```diff
@@ -1,222 +1,148 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  FormEvent,
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import { createTodo, deleteTodos, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components_Todo/ErrorNotification';
+import { NewTodo } from './components_Todo/NewTodo';
+import { TodoFilter } from './components_Todo/TodoFilter';
+
+import { TodoList } from './components_Todo/TodoList';
+import { FilterStatus } from './types/FilterStatus';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [newTitleTodo, setNewTitleTodo] = useState('');
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoId, setTodoId] = useState([0]);
+
+  const [statusPatch, setStatusPatch] = useState('');
+  const [statusFilter, setStatusFilter] = useState('all');
+  const [hasLoadError, setHasLoadError] = useState('');
+  const [isAdding, setIsAdding] = useState(false);
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
+
+    if (user) {
+      setIsAdding(true);
+      getTodos(user.id)
+        .then(todo => {
+          setTodos(todo);
+        }).catch(() => (
+          setHasLoadError('Unable to load a todo')
```

Comment:
Do not rely on the unknown string, make constants for this. Better crate enum for this purpose and use it all over the code when you work with errors. When you would write test tasks, using `enum` in similar case would be a ➕